src/pages/Onboard/NameTeam.vue

```diff
@@ -130,18 +136,92 @@ export default {
       await this.createLicense()
       if (!this.updateServerError) this.goToResources()
     },
-    goToResources() {
+    async accept(pt) {
+      try {
+        this.accepting = true
+        const invitationId = pt.id
+        const accepted = await this.acceptMembershipInvitation(invitationId)
+        this.redirectTenant = pt.tenant.slug
+        if (accepted.accept_membership_invitation.id) {
+          this.redirectTenant = pt.tenant.slug
+          this.accepting = false
+        }
+      } catch (e) {
+        this.mutationErrorMessage = e
+          .toString()
+          .split(':')
+          .pop()
+        this.loading = false
+        this.error = true
+      }
+      this.loading = false
+    },
+    async decline(id) {
+      try {
+        this.deleting = true
+        const declined = await this.declineMembershipInvitation(id)
+        if (declined.delete_membership_invitation.success) {
+          this.$emit('hide')
+        }
+        this.deleting = false
+      } catch (e) {
+        this.mutationErrorMessage = e
+          .toString()
+          .split(':')
+          .pop()
```

Comment:
Is there a reason to only return the final portion of the error message? I notice we're doing the same above.

Reply:
@zhen0 this was code I took from your accept invitation work, what was the logic in only using the end of the error message?

Reply:
Most error messages from the backend come in the format "Error: Error doing something......."  E.g. "Error: Error refreshing token in prefectRefresh". The formatting was to remove the double "Error".